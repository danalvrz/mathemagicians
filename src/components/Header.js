import React from 'react';
import { Link } from 'react-router-dom';

function Header() {
  return (
    <nav className="menuContainer">
      <h1>Mathemagicians</h1>
      <ul className="menuList">
        <li><Link to="/" style={{ color: 'aliceblue' }}>Home</Link></li>
        <li><Link to="/calculator" style={{ color: 'aliceblue' }}>Cowculator</Link></li>
        <li><Link to="/quote" style={{ color: 'aliceblue' }}>Quote</Link></li>
      </ul>
    </nav>
  );
}

export default Header;
