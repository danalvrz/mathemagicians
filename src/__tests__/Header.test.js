import React from 'react';
import renderer from 'react-test-renderer';
// eslint-disable-next-line no-unused-vars
import { render, fireEvent } from '@testing-library/react';
import { HashRouter as Router } from 'react-router-dom';
import Header from '../components/Header';

test('Header', () => {
  const component = renderer.create(
    <Router>
      <Header />
    </Router>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
