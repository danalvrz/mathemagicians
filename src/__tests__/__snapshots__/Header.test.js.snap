// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderTesting Header snapshot 1`] = `
<header
  className="menuContainer"
>
  <h1>
    Mathemagicians
  </h1>
  <nav
    className="menuList"
  >
    <a
      href="#/"
      onClick={[Function]}
    >
      Home
    </a>
    |
    <a
      href="#/calculator"
      onClick={[Function]}
    >
      Cowculator
    </a>
    |
    <a
      href="#/quote"
      onClick={[Function]}
    >
      Quote
    </a>
  </nav>
</header>
`;
